src/views/open-api-docs-view/components/operation-header/operation-header.module.css

```
@@ -0,0 +1,22 @@
+.wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+}
+
+.heading {
+	composes: g-offset-scroll-margin-top from global;
+	composes: hds-typography-display-400 from global;
+	font-weight: var(--token-typography-font-weight-bold);
+}
+
+.methodAndPath {
+	display: flex;
+	align-items: center;
+	gap: 8px;
+}
+
+.path {
+	composes: hds-typography-code-300 from global;
+	color: var(--token-color-foreground-primary);
```

Quick fix for overflow issues on mobile:

```suggestion
	color: var(--token-color-foreground-primary);
	overflow-wrap: break-word;
```